event.go

```diff
@@ -234,40 +327,69 @@ type SendEvent struct {
 	Device string `json:"device,omitempty"`
 
 	// A json object
-	Payload *SendEventPayload `json:"payload,omitempty"`
+	Payload Payload `json:"payload,omitempty"`
```

Comment:
And do why switch to this empty `struct` here?

Reply:
What do you think about leaving it as `any` and remove the struct?

Reply:
Or event better `map[string]any`

Reply:
Okay for me, left it just to have common interface, but for now it not important I think.

Will do update soon

Reply:
@SkYNewZ It requires also go update to at least 1.18.

Probably it okay to update go version to the latest?

Reply:
You can keep it as `interface{}`. Maybe I will upgrade in a different pull request.

Reply:
Done